llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2020,6 +2032,8 @@ class VPReductionPHIRecipe : public VPHeaderPHIRecipe {
     return R->getVPDefID() == VPDef::VPReductionPHISC;
   }
 
+  void SetVFScaleFactor(unsigned ScaleFactor) { VFScaleFactor = ScaleFactor; }
```

Comment:
This won't be needed if `postInsertionOp` is removed.

Reply:
Done.